news/2015-05-08-workshop-business-advantages.html

```diff
@@ -108,9 +108,11 @@ <h3>Workshop: The Business Advantages of Making Special the Norm</h3>
                     <a href="http://idrc.ocadu.ca/">Inclusive Design Research Centre (IDRC)</a>
                     and professor at <a href="http://ocadu.ca">OCAD University</a> in Toronto.</p>
                 <p>For more information or to register, visit <a href="http://www.viscardicenter.org/news-events/events/nbdc-distance-learning.html">http://www.viscardicenter.org/news-events/events/nbdc-distance-learning.html</a>.</p>
-            </div>
 
-            <div class="floe-footer">
+                <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/cjXKHzcQypw" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
Is there a reason why we're using the nocookie site instead of the full YouTube site?

Reply:
It was an option from youtube. It's supposed to not track the user unless they actually watch the video.